.jenkins/pytorch/win-test.sh

```diff
@@ -78,14 +78,37 @@ call ci_scripts/setup_pytorch_env.bat
 cd test/ && python run_test.py --exclude nn --verbose && cd ..
 EOL
 
+cat >ci_scripts/test_custom_script_ops.bat <<EOL
+call ci_scripts/setup_pytorch_env.bat
+
+cd test/custom_operator
+
+:: Build the custom operator library.
+mkdir build
+cd build
+:: Note: Caffe2 does not support MSVC + CUDA + Debug mode (has to be Release mode)
+cmake -DCMAKE_PREFIX_PATH="..\\torch" -DCMAKE_BUILD_TYPE=Release -DUSE_MKL=OFF -G"NMake Makefiles" ..
```

Comment:
Err... can we please run builds in `win-build.sh`? (We had another incident like this with the OS X builds and it is very annoying when the builds get delayed like this!)

Reply:
Yes that's a good idea. I'll need to figure out where to place the build artifacts so that they get zipped correctly. I'll wait until we figure out how to make it compile on Windows at all first.

Reply:
I don't think it's Caffe2's problem regarding MSVC + cuda + debug: if msvc now supports it, we should be able to fully support it now as well.

Reply:
I remember running into an error with that configuration two weeks ago. Did this change recently?